_test/test_gen_sqw_for_powders/test_combine_cyl.m

```diff
@@ -68,30 +68,30 @@
             % Handle class! test files are in SVN
             %add_to_files_cleanList(this,this.spe_file_1,this.spe_file_2);
             add_to_path_cleanList(this,test_functions_path);
-            
-            
+
+
         end
-        
+
         function this=test_combine_pow1(this)

```

Comment:
why is this `test_combine_pow1` when it calls cylinder test helper? Should this be `test_combine_cyl1` to match the file name in line 78?